src/Kdyby/Translation/Translator.php

```diff
@@ -79,12 +85,13 @@ class Translator extends BaseTranslator implements ITranslator
 	 * @param IResourceLoader $loader
 	 */
 	public function __construct(IUserLocaleResolver $localeResolver, MessageSelector $selector,
-		CatalogueCompiler $catalogueCompiler, FallbackResolver $fallbackResolver, IResourceLoader $loader)
+		CatalogueCompiler $catalogueCompiler, FallbackResolver $fallbackResolver, IResourceLoader $loader, ILogger $logger)
```

Comment:
Well, you know, this is Nette extension, sou Tracy shouldn't be such a burden

Could it implement both and on the app level decide which it should use with priority on LoggerInterface instead of ILogger ?

Missing message would then end up in warning loglevel?